fiatconnect-api.md

```diff
@@ -1796,6 +1796,293 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+####  7.3.2.2. `CheckingAccount`
+
+`CheckingAccountInr` is a Fiat Account schema that represents accounts for transactions across multiple currencies.
+
+```
+{
+    "accountName": `string`,
+    "institutionName": `string`,
+    "institutionCode": `string`,
+    "accountNumber": `string`,
+    "fiatAccountType": `FiatAccountTypeEnum.BankAccount`
+}
+```
+
+The `country` field should be a [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code. The field is for providers to specify which country this data is meant for.
+
+Based on the country `institutionName` and `institutionCode` should be supplied. AllowedValues Enum is mentioned in section 7.3.2.2.1 
```

Comment:
As we discussed, could we split this schema out into separate schemas? Right now, this schema is overloaded between representing *both* bank transfers and mobile transfers. 

We already have a [MobileMoney](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#7322-mobilemoney) Fiat Account schema; I think you could probably re-use this for mobile payments. Instead of having a user select a machine-readable "mobile provider code", you could have them select a human-readable/parsable string from the `SupportedOperatorEnum` list. 

For bank transfers, it sounds like you could also have two separate schemas: one for bank accounts where IFSC code is required, and another for bank accounts where IBAN is required.

Just to be clear, the reason I'm strongly in favor of having multiple schemas for each different type of account is due to the end-user experience we'll be able to present to the user. If we just have one huge, monolithic schema that includes *all* different account types, it becomes extremely difficult to prompt the user for the correct information. From a client developer's perspective, using one monolithic schema makes it significantly more difficult to implement a form or UI to prompt the user for correct information in a way that is intelligible and makes sense to the end user.

To reiterate; I think my recommendation here would be to:
1. Try to re-use the `MobileMoney` schema for mobile transfers. You can add to the `SupportedOperatorEnum` list for enumerating allowed providers by setting the `allowedValues` field in the quote response. In the backend, you can perform a mapping from the selected provider string to the code that you require.
2. Add a new schema for IFSC bank accounts. You may actually be able to re-use the [AccountNumber](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#7321-accountnumber) Fiat Account schema, with the stipulation that when `country` is set to `"IN"`, `accountNumber` adheres to IFSC syntax. I would try to re-use this schema if possible.
3. Add a new schema for IBAN accounts. Since IBAN codes are internationally recognizable, IBAN-based fiat account information belongs in its own schema, rather than re-using the `AccountNumber` schema.